test/integration/controlplane/transformation/transformation_test.go

```diff
@@ -550,7 +551,7 @@ func (e *transformTest) printMetrics() error {
 func mustBeHealthy(t kubeapiservertesting.Logger, checkName, wantBodyContains string, clientConfig *rest.Config, excludes ...string) {
 	t.Helper()
 	var restErr error
-	pollErr := wait.PollImmediate(1*time.Second, 5*time.Minute, func() (bool, error) {
+	pollErr := wait.PollUntilContextTimeout(context.TODO(), 1*time.Second, 5*time.Minute, true, func(ctx context.Context) (bool, error) {
```

Comment:
What is the purpose of this change?

Reply:
pull-kubernetes-linter-hints failed due to the deprecation of PollImmediate and suggested to use PollUntilContextTimeout